plugins/notification-resources/src/components/EmployeeInbox.svelte

```diff
@@ -167,6 +158,15 @@
       changeSelected(selected)
     }
   }
+
+  let dmInput: AnySvelteComponent | undefined = undefined
```

Comment:
I've expected it be like Employee InboxFooter, so I could contribute a component into. Let's think about it on next PR.